### DISCUSSION TITLE
Docker v.4.33.0 can't connect to database

### DISCUSSION THREAD

**Discussion by sam-egrv** [wardenenv/warden#791] [Help & Support]
### Version of Warden

0.14.3

### Operating System and Installation Method

Mac OS 13.4 via github

### Image

maria-db:10.6-magento2

### Describe the Bug

After updated my Docker Desktop to v.4.33.0 I can't connect to DB neither via PhpStorm or Sequel Pro.
I was able to connect to the tunnel.warden.test via terminal or PhpStorm. But connection to DB doesn't work.
Here is my config in PhpStorm:
<img width="824" alt="ssh_config" src="https://github.com/user-attachments/assets/88006a91-3433-4b19-9619-798f0a6dfc68">
But when I'm trying to connect I got "SSH tunnel creation failed: Name does not resolve.":
<img width="802" alt="DB config" src="https://github.com/user-attachments/assets/673ae7e1-9e44-4ac5-827a-5ceba1d3a335">

Solution for the similar issue didn't help: https://github.com/wardenenv/warden/issues/260#issuecomment-722355346

**Comment by navarr** [comment]
Could you please show your general tab? Chiefly the hostname that you're attempting to connect to
